src/Theme/defaultTheme.js

```diff
@@ -16,12 +16,13 @@ export default {
     light: '#e57373',
     dark: '#d32f2f',
   },
-  text: {
+  textColor: {
```

Comment:
This theming is never used, should it be applied to `BaseText` based on state?

Reply:
Yeah this will be handled in https://github.com/codypearce/material-bread/issues/191